app/controllers/admin/enterprises_controller.rb

```diff
@@ -198,6 +205,14 @@ def update_tag_rules(tag_rules_attributes)
       end
     end
 
+    def update_enterprise_notifications
+      notify_user = params[:receives_notifications]
+      if notify_user.present?
+        EnterpriseRole.set_notification_user notify_user, @object.id
+        params.delete :receives_notifications
```

Comment:
why do we need to delete it? Will it affect some other part of the code down the line?